### ISSUE TITLE
False negative - Credentials transported over the network

### ISSUE THREAD

**Issue by dio-cmx** [42Crunch/vscode-openapi#26]
Check shows negative "Credentials transported over the network" when using client-id+client-secret or basic authentication over HTTPS
Description says that
> The scheme is set to **http**...

, that is wrong

**Comment by ak1394** [comment]
Thanks a lot for reporting! Is it v3 or v2 swagger you're using? Could you provide a sample of openapi file?

**Comment by dio-cmx** [comment]
```
{
    "swagger": "2.0",
    "info": {
        "title": "API ",
        "version": "0.1.0"
    },
    "schemes": [
        "https"
    ],
    "host": "api.company.ru",
    "basePath": "/api/v1",
    "consumes": [
        "application/json"
    ],
    "produces": [
        "application/json"
    ],
    "securityDefinitions": {
        "clientIdHeader": {
            "type": "apiKey",
            "in": "header",
            "name": "X-IBM-Client-Id"
        },
        "clientSecretHeader": {
            "type": "apiKey",
            "in": "header",
            "name": "X-IBM-Client-Secret"
        }
    },
    "security": [
        {
            "clientIdHeader": [],
            "clientSecretHeader": []
        }
    ],
    "paths": {
        "/test": {
            "post": {
                "responses": {
                    "201": {
                        "description": "201 Created"
                    },
                    "400": {
                        "description": "400 Bad request"
                    }
                },
                "parameters": [
                    {
                        "name": "body",
                        "required": true,
                        "in": "body",
                        "schema": {
                            "type": "object"
                        }
                    }
                ]
            }
        }
    }

}
```

**Comment by ak1394** [comment]
Fixed in 2.0.1. The fix related to the wording of the warning. Previously it was mistakingly talking about credentials being sent over http, now warning suggests to use oauth.
